src/System.Reflection/tests/FieldInfo/FieldInfo_CustomAttributeTests.cs

```diff
@@ -41,79 +35,32 @@ public int Prop
     // Test class for Custom Attribute Test
     public class FieldInfoCustomAttributeTests
     {
-        //Test for custom Attribute of type  Int32AttrSimple
-        [Fact]
-        public void Test_Int32AttrSimple()
+        public static IEnumerable<object[]> FieldInfoCustomAttributes_TestData()
```

Comment:
Another case where it looks like this could be done as InlineData instead of MemberData.  I'll stop commenting on these.